src/arrow.js

```diff
@@ -93,4 +96,23 @@ export default class Arrow {
         this.calculate_path();
         this.element.setAttribute('d', this.path);
     }
+    
+    //  add event handling for Arrows
+    setup_eventListener(){
```

Comment:
Please use snake case, `setup_event_listener`. Also, this should be called `setup_dependency_editing`